src/app/form/(formSteps)/personal-details/2/PersonalDetailsStep2.test.tsx

```diff
@@ -205,4 +205,38 @@ describe("<PersonalDetailsStep2 />", () => {
     expect(screen.getByRole("combobox", { name: "State *" })).toHaveValue("NJ");
     expect(screen.getByRole("textbox", { name: "ZIP code *" })).toHaveValue("12345");
   });
+
+  describe("Public information explainer", () => {
+    it("orders the public information explainer immediately after the address input", async () => {
+      const user = userEvent.setup();
+      renderWithRouter();
+
+      const zipCodeInput = screen.getByRole("textbox", {
+        name: "ZIP code *",
+      });
+      const publicInformationExplainer = screen.getByRole("button", {
+        name: "Will my information be public?",
+      });
+      await user.type(zipCodeInput, "1");
+      expect(zipCodeInput).toHaveFocus();
+
+      await user.tab();
+      expect(publicInformationExplainer).toHaveFocus();
+    });
+
+    it("has a heading level one greater than the section heading level", () => {
+      renderWithRouter();
```

Comment:
Checking heading level by checking the level of the heading it's nested under is really neat! It also seems like a GREAT thing to abstract to a reusable util and makes me dream of an OOI library of testing library extensions like this